docs/includes/yo-office-command-guidance.md

```diff
@@ -0,0 +1,10 @@
+Run the following command to create an add-in project using the Yeoman generator: 
+
+```command&nbsp;line
+yo office
+```
+
+> [!NOTE]
+> When you run the `yo office` command, you may receive prompts about the data collection policies of Yeoman and the Office Add-in CLI tools. Use the information that's provided to respond to the prompts as you see fit. If you choose **Exit** in response to the second prompt, you'll need to run the `yo office` command again when you're ready to create your add-in project.
```

Comment:
It feels weird to specify the "second prompt" when they might not show up. Also, are we worried that people are going to be confused that exiting yo office ruins the tutorial? That seems intuitive to me.

Reply:
@AlexJerabek Valid points -- thanks for the feedback! I'll remove the last sentence from this note (starting with "If you choose Exit..."), and will make the same change to this verbiage in OfficeDev/office-js-docs-pr.